android/app/proguard-rules.pro

```diff
@@ -0,0 +1,71 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
+-optimizationpasses 5
+-dontusemixedcaseclassnames
+-dontskipnonpubliclibraryclasses
+-dontpreverify
+-dontshrink
+-verbose
+-ignorewarnings
+
+-keeppackagenames io.scanbot.barcodescanner.**
+-keep public class io.scanbot.barcodescanner.**{ *; }
+
+-keeppackagenames io.scanbot.sdk.barcode.**
+-keep public class io.scanbot.sdk.barcode.** { *; }
+
+-keeppackagenames io.scanbot.sdk.barcode_scanner.**
+-keep public class io.scanbot.sdk.barcode_scanner.** { *; }
+
+-keeppackagenames io.scanbot.barcode.sdk.flutter.**
+-keep public class io.scanbot.barcode.sdk.flutter.**{ *; }
+
+-keeppackagenames io.scanbot.sap.**
+-keep public class io.scanbot.sap.** { *; }
+
+-keepclassmembers enum io.scanbot.** {*;}
+-keepclassmembers class io.scanbot.** extends java.lang.Enum {
+    <fields>;
+    public static **[] values();
+    public static ** valueOf(java.lang.String);
+}
+
+# JSR 305 annotations are for embedding nullability information.
+-dontwarn javax.annotation.**
```

Comment:
it would be great to live gson comments, so the config would be unified with bundle sdk